examples/vulkan/Vulkan.cpp

```diff
@@ -1445,7 +1446,7 @@ class VulkanExample
         void* ptr = nullptr;
 
         // Map the buffer into our address space
-        if (vkMapMemory(device, stagingBufferMemory, 0, sizeof(vertexData), 0, &ptr) != VK_SUCCESS)
```

Comment:
This is not correct, this gives the vertex count but not the size of the vertex this should be multiplied by sizeof(vertexData[0]); or continue using sizeof